clients/roscpp/include/ros/transport_hints.h

```diff
@@ -105,6 +105,31 @@ class ROSCPP_DECL TransportHints
     return false;
   }
 
+  /**
+   * \brief Set TOS level for traffic on this connection.
+   *
+   * \param ip_tos Level of TOS to set - see <netinet/ip.h>
+   */
+  TransportHints& ipTos(int ip_tos)
+  {
+    options_["ip_tos"] = boost::lexical_cast<std::string>(ip_tos);
+    return *this;
+  }
+
+  /**
+   * \brief Returns whether or not this TransportHints has specified IP_TOS level.
+   */
+  bool getIpTos()
+  {
+    M_string::iterator it = options_.find("ip_tos");
+    if (it == options_.end())
+    {
+      return 0;
```

Comment:
```suggestion
      return false;
```

Reply:
This should not return a bool, oops. Copy/paste error

Reply:
In that case, is there an enum for `0`?